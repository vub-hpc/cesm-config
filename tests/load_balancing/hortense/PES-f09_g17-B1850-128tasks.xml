<?xml version="1.0"?>
<config_pes>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nS">
        <comment>Sequential</comment>
        <ntasks>
          <ntasks_atm>128</ntasks_atm>
          <ntasks_lnd>128</ntasks_lnd>
          <ntasks_rof>128</ntasks_rof>
          <ntasks_ice>128</ntasks_ice>
          <ntasks_ocn>128</ntasks_ocn>
          <ntasks_glc>128</ntasks_glc>
          <ntasks_wav>128</ntasks_wav>
          <ntasks_cpl>128</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>0</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25">
        <comment>Concurrent OCN at 25 percent</comment>
        <ntasks>
          <ntasks_atm>96</ntasks_atm>
          <ntasks_lnd>96</ntasks_lnd>
          <ntasks_rof>96</ntasks_rof>
          <ntasks_ice>96</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>96</ntasks_glc>
          <ntasks_wav>96</ntasks_wav>
          <ntasks_cpl>96</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>96</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO50">
        <comment>Concurrent OCN at 50 percent</comment>
        <ntasks>
          <ntasks_atm>64</ntasks_atm>
          <ntasks_lnd>64</ntasks_lnd>
          <ntasks_rof>64</ntasks_rof>
          <ntasks_ice>64</ntasks_ice>
          <ntasks_ocn>64</ntasks_ocn>
          <ntasks_glc>64</ntasks_glc>
          <ntasks_wav>64</ntasks_wav>
          <ntasks_cpl>64</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>0</rootpe_ice>
          <rootpe_ocn>64</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nI50L50">
        <comment>Concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>128</ntasks_atm>
          <ntasks_lnd>64</ntasks_lnd>
          <ntasks_rof>128</ntasks_rof>
          <ntasks_ice>64</ntasks_ice>
          <ntasks_ocn>128</ntasks_ocn>
          <ntasks_glc>128</ntasks_glc>
          <ntasks_wav>128</ntasks_wav>
          <ntasks_cpl>128</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>64</rootpe_ice>
          <rootpe_ocn>0</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I50L50">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>96</ntasks_atm>
          <ntasks_lnd>48</ntasks_lnd>
          <ntasks_rof>96</ntasks_rof>
          <ntasks_ice>48</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>96</ntasks_glc>
          <ntasks_wav>96</ntasks_wav>
          <ntasks_cpl>96</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>48</rootpe_ice>
          <rootpe_ocn>96</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I75L25">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 75/25 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>96</ntasks_atm>
          <ntasks_lnd>32</ntasks_lnd>
          <ntasks_rof>96</ntasks_rof>
          <ntasks_ice>64</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>96</ntasks_glc>
          <ntasks_wav>96</ntasks_wav>
          <ntasks_cpl>96</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>32</rootpe_ice>
          <rootpe_ocn>96</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO25I25L75">
        <comment>Concurrent OCN at 25 percent with concurrent ICE/LND at 25/75 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>96</ntasks_atm>
          <ntasks_lnd>64</ntasks_lnd>
          <ntasks_rof>96</ntasks_rof>
          <ntasks_ice>32</ntasks_ice>
          <ntasks_ocn>32</ntasks_ocn>
          <ntasks_glc>96</ntasks_glc>
          <ntasks_wav>96</ntasks_wav>
          <ntasks_cpl>96</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>64</rootpe_ice>
          <rootpe_ocn>96</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
  <grid name="any">
    <mach name="any">
      <pes compset="any" pesize="4nO50I50L50">
        <comment>Concurrent OCN at 50 percent with concurrent ICE/LND at 50/50 in sequence with ATM</comment>
        <ntasks>
          <ntasks_atm>64</ntasks_atm>
          <ntasks_lnd>32</ntasks_lnd>
          <ntasks_rof>64</ntasks_rof>
          <ntasks_ice>32</ntasks_ice>
          <ntasks_ocn>64</ntasks_ocn>
          <ntasks_glc>64</ntasks_glc>
          <ntasks_wav>64</ntasks_wav>
          <ntasks_cpl>64</ntasks_cpl>
        </ntasks>
        <nthrds>
          <nthrds_atm>1</nthrds_atm>
          <nthrds_lnd>1</nthrds_lnd>
          <nthrds_rof>1</nthrds_rof>
          <nthrds_ice>1</nthrds_ice>
          <nthrds_ocn>1</nthrds_ocn>
          <nthrds_glc>1</nthrds_glc>
          <nthrds_wav>1</nthrds_wav>
          <nthrds_cpl>1</nthrds_cpl>
        </nthrds>
        <rootpe>
          <rootpe_atm>0</rootpe_atm>
          <rootpe_lnd>0</rootpe_lnd>
          <rootpe_rof>0</rootpe_rof>
          <rootpe_ice>32</rootpe_ice>
          <rootpe_ocn>64</rootpe_ocn>
          <rootpe_glc>0</rootpe_glc>
          <rootpe_wav>0</rootpe_wav>
          <rootpe_cpl>0</rootpe_cpl>
        </rootpe>
      </pes>
    </mach>
  </grid>
</config_pes>
