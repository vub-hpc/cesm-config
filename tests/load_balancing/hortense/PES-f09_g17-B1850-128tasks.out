found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing
found /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I0.f09_g17.B1850.hortense_gnu.lbt.1833679.220921-103004
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I0.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833679.220921-103004
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833680.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I1.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I1.f09_g17.B1850.hortense_gnu.lbt.1833680.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833681.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I2.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I2.f09_g17.B1850.hortense_gnu.lbt.1833681.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833682.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I3.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I3.f09_g17.B1850.hortense_gnu.lbt.1833682.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833683.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I4.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I4.f09_g17.B1850.hortense_gnu.lbt.1833683.220921-105057
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I5.f09_g17.B1850.hortense_gnu.lbt.1833685.220921-112124
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I5.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833685.220921-112124
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I6.f09_g17.B1850.hortense_gnu.lbt.1833687.220921-114216
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I6.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833687.220921-114216
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing.PFS_I7.f09_g17.B1850.hortense_gnu.lbt.1833688.220921-114239
Reading timing file /dodrio/scratch/users/vsc10122/cesm/output/PFS_I7.f09_g17.B1850.hortense_gnu.lbt/timing/cesm_timing_stats.1833688.220921-114239
Using layout  = IceLndWavAtmOcn
GLPSOL--GLPK LP/MIP Solver 5.0
Parameter(s) specified in the command line:
 --cpxlp /tmp/bf01f8ee2b7e41f287c58816b0683f90-pulp.lp -o /tmp/bf01f8ee2b7e41f287c58816b0683f90-pulp.sol
Reading problem data from '/tmp/bf01f8ee2b7e41f287c58816b0683f90-pulp.lp'...
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
58 lines were read
GLPK Integer Optimizer 5.0
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
Preprocessing...
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
Scaling...
 A: min|aij| =  2.872e-01  max|aij| =  3.845e+03  ratio =  1.339e+04
GM: min|aij| =  2.063e-01  max|aij| =  4.848e+00  ratio =  2.350e+01
EQ: min|aij| =  4.255e-02  max|aij| =  1.000e+00  ratio =  2.350e+01
2N: min|aij| =  3.125e-02  max|aij| =  1.278e+00  ratio =  4.091e+01
Constructing initial basis...
Size of triangular part is 31
Solving LP relaxation...
GLPK Simplex Optimizer 5.0
31 rows, 17 columns, 65 non-zeros
      0: obj =   2.187999000e+03 inf =   1.410e+03 (17)
     23: obj =   4.366381594e+03 inf =   4.322e-14 (0)
*    25: obj =   4.317284483e+03 inf =   0.000e+00 (0)
OPTIMAL LP SOLUTION FOUND
Integer optimization begins...
Long-step dual simplex will be used
+    25: mip =     not found yet >=              -inf        (1; 0)
+    28: >>>>>   4.662988000e+03 >=   4.662988000e+03   0.0% (4; 0)
+    28: mip =   4.662988000e+03 >=     tree is empty   0.0% (0; 7)
INTEGER OPTIMAL SOLUTION FOUND
Time used:   0.0 secs
Memory used: 0.1 Mb (92945 bytes)
Writing MIP solution to '/tmp/bf01f8ee2b7e41f287c58816b0683f90-pulp.sol'...
Solving Mixed Integer Linear Program using PuLP interface to COIN-CBC
GLPSOL--GLPK LP/MIP Solver 5.0
Parameter(s) specified in the command line:
 --cpxlp /tmp/0bad00720fda49c8b92ec70a02521966-pulp.lp -o /tmp/0bad00720fda49c8b92ec70a02521966-pulp.sol
Reading problem data from '/tmp/0bad00720fda49c8b92ec70a02521966-pulp.lp'...
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
58 lines were read
GLPK Integer Optimizer 5.0
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
Preprocessing...
31 rows, 17 columns, 65 non-zeros
10 integer variables, none of which are binary
Scaling...
 A: min|aij| =  2.872e-01  max|aij| =  3.845e+03  ratio =  1.339e+04
GM: min|aij| =  2.063e-01  max|aij| =  4.848e+00  ratio =  2.350e+01
EQ: min|aij| =  4.255e-02  max|aij| =  1.000e+00  ratio =  2.350e+01
2N: min|aij| =  3.125e-02  max|aij| =  1.278e+00  ratio =  4.091e+01
Constructing initial basis...
Size of triangular part is 31
Solving LP relaxation...
GLPK Simplex Optimizer 5.0
31 rows, 17 columns, 65 non-zeros
      0: obj =   2.187999000e+03 inf =   1.410e+03 (17)
     23: obj =   4.366381594e+03 inf =   4.322e-14 (0)
*    25: obj =   4.317284483e+03 inf =   0.000e+00 (0)
OPTIMAL LP SOLUTION FOUND
Integer optimization begins...
Long-step dual simplex will be used
+    25: mip =     not found yet >=              -inf        (1; 0)
+    28: >>>>>   4.662988000e+03 >=   4.662988000e+03   0.0% (4; 0)
+    28: mip =   4.662988000e+03 >=     tree is empty   0.0% (0; 7)
INTEGER OPTIMAL SOLUTION FOUND
Time used:   0.0 secs
Memory used: 0.1 Mb (92945 bytes)
Writing MIP solution to '/tmp/0bad00720fda49c8b92ec70a02521966-pulp.sol'...
PuLP solver status: Solved
COST_ATM = 2617.640000
COST_ICE = 2045.350000
COST_LND = 742.861000
COST_OCN = 4662.990000
COST_TOTAL = 4662.990000
COST_WAV = 2045.350000
NBLOCKS_ATM = 12
NBLOCKS_ICE = 3
NBLOCKS_LND = 4
NBLOCKS_OCN = 4
NBLOCKS_WAV = 5
NTASKS_ATM = 96
NTASKS_ICE = 24
NTASKS_LND = 32
NTASKS_OCN = 32
NTASKS_TOTAL = 128
NTASKS_WAV = 40
